Lib/logging/config.py

```diff
@@ -694,7 +694,11 @@ def add_filters(self, filterer, filters):
         """Add filters to a filterer from a list of names."""
         for f in filters:
             try:
-                filterer.addFilter(self.config['filters'][f])
+                if callable(f) or callable(getattr(f, "filter", None)):
+                    filter_ = f
```

Comment:
Any particular reason for using `filter_` rather than just `filter`? I normally only use trailing underscores when the word would otherwise be a keyword (e.g. `class_`, `async_`).

Reply:
Trying not to hide builtin [`filter`](https://docs.python.org/3/library/functions.html#filter). It is not a big deal but this avoids that in the future we remove the variable declaration we will be forced to delete the removal as well.

If this was named just `filter`, one could by mistake delete line 698 and everything would "continue to work" as we will be passing the `filter` builtin when we do not redefine it.